phys/module_mp_jensen_ishmael.F

```diff
@@ -529,100 +552,18 @@ subroutine newmicro(it, kts, kte, i, j, dt, pres_e, dzmic,    &
     
   !.. Some dummy variables
     real :: dum, dum1, dumt, dumgam, dumgam2
+    real :: tmpsum, tmpsumr, tmpmelt
```

Comment:
@graupely 
Are these needed? If so, can the tmpsum,tmpsumr,tmpmelt combo be moved into the PR with the substantive changes?

Reply:
@davegill, 
They are needed. They were in a different location in the original pull, I moved them to be in the dummy variable section.

Reply:
Gotcha